### DISCUSSION TITLE
Lrzip-next and zpaqfranz

### DISCUSSION THREAD

**Discussion by demhademha** [pete4abw/lrzip-next#95] [Q&A]
Would it be possible to use lrzip-next and https://github.com/fcorbelli/zpaqfranz to form incremental backups
Thank you

**Comment by pete4abw** [comment]
Hi @demhademha and thanks for the suggestion . I've looked at this. `lrzip-next` is not a journaling archiver. `lrzip-next` passes data already received to a compression backend. So attempting any incremental work after this is not possible. It's all or nothing. Using a front-end like `tar` or `find` and piping that output to `lrzip-next` is the way to go.

**Reply by demhademha** [reply]
How and why would find and lrzip-next be used together

**Reply by pete4abw** [reply]
I would not use it, but it's a possibility. Let's say you had a small group of files you wanted to compress, you could use something like

`find . -type f -name 'forcompression*' -exec 'lrzip-next '{}' \;`

with additional options possible. It _could_ be useful if files are in different directories because find works recursively in directories.

Of course, `lrzip-next forcompression*` would work too, but only in one directory.
